src/astro/sql/operators/temp_hooks.py

```diff
@@ -56,3 +61,46 @@ def get_uri(self) -> str:
         if self.schema:
             uri += self.schema
         return uri
+
+
+class TempBigQueryHook(BigQueryHook):
```

Comment:
@utkarsharma2 can you please document the bug you are working around so we can remove this once the upstream is fixed?

Reply:
Sure, will add a comment.